packs/hpe-icsp/actions/lib/icsp.py

```diff
@@ -0,0 +1,98 @@
+# Licensed to the StackStorm, Inc ('StackStorm') under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import requests
+from st2actions.runners.pythonrunner import Action
+
+
+class ICSPBaseActions(Action):
+
+    def __init__(self, config):
+        super(ICSPBaseActions, self).__init__(config)
+        self.icsp_host = config['host']
+        self.icsp_user = config['user']
+        self.icsp_pass = config['pass']
+        self.icsp_apiv = config['apiv']
+        self.icsp_sslverify = config['sslverify']
+
+    def checkResults(self, results):
```

Comment:
Minor style / consistency things - we usually use underscore notation for method names (`check_results`, `set_connection`, etc.).